Source/art/ftfont.m

```diff
@@ -22,17 +22,6 @@
    Boston, MA 02110-1301, USA.
 */
 
-#include <ft2build.h>
-#include FT_FREETYPE_H
-
-
-#if (FREETYPE_MAJOR==2) && ((FREETYPE_MINOR<1) || ((FREETYPE_MINOR==1) && (FREETYPE_PATCH<8)))
-
-#import "ftfont-old.m"
```

Comment:
If we no longer include the file here, we might as well delete it.

Reply:
I'm waiting for Riccardo results.

Reply:
I think the issues I am having are not related with the removal of ftflond-old.m, the freetype release I am using is newer than 2.1.1 on my test machines and the code compiles